beard/utils/joblib_helpers.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Beard.
+# Copyright (C) 2015 CERN.
+#
+# Beard is a free software; you can redistribute it and/or modify it
+# under the terms of the Revised BSD License; see LICENSE file for
+# more details.
+
+"""Joblib library utils.
+
+.. codeauthor:: Mateusz Susik <mateusz.susik@cern.ch>
+
+"""
+
+from joblib import cpu_count
+
+
+def jobs_from_njobs(n_jobs):
+    """Compute number of jobs from Joblib's n_jobs variable.
+
+    Parameters
+    ----------
+
+    :param n_jobs: integer
+        The value used by Joblib library
+
+    Returns
+    -------
+    :returns: integer
+        The number of jobs which would be created if ``n_jobs`` was used
+        by Joblib.
+    """
+    return cpu_count() + 1 + n_jobs if n_jobs == -1 else n_jobs
```

Comment:
Will be changed to
` return cpu_count() + 1 + n_jobs if n_jobs < 0 else n_jobs `